### PR TITLE
Add info about Vue.js integration to README file

### LINKED ISSUE #4089: Mention 'Vue' in README.md
## Type of report

Task

## Provide a description of requested docs changes

As the Vue integration is officially stable, there is no reason not to mention it [here](https://github.com/ckeditor/ckeditor4#integrating-with-angular-and-react) 🎉

### PR DESCRIPTION
**Pull request by Dumluregn**
## What is the purpose of this pull request?

Taskk

## Does your PR contain necessary tests?

All patches that change the editor code must include tests. You can always read more
on [PR testing](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_contributing_code.html#tests),
[how to set the testing environment](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html) and
[how to create tests](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html#creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [ ] Unit tests
- [ ] Manual tests

## Did you follow the CKEditor 4 code style guide?

Your code should follow the guidelines from the [CKEditor 4 code style guide](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md) which helps keep the entire codebase consistent.

- [ ] PR is consistent with the code style guide

## What is the proposed changelog entry for this pull request?

skip

## What changes did you make?

Added Vue.js mention and link to npm package.

## Which issues does your PR resolve?

Closes #4089.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,9 +50,9 @@ Load the CKEditor 4 script from CDN:
 </script>
 ```
 
-### Integrating with Angular and React
+### Integrating with Angular, React and Vue.js
```
Comment (f1ames) [inline README.md:53]: TBH I'm not sure if it should be `Vue.js` or just `Vue`? In most places in docs we mention it as `Vue` (except for https://ckeditor.com/docs/ckeditor4/latest/guide/index.html where `Vue.js` is used) :thinking: 

Looking at https://www.npmjs.com/package/vue it seems official project name is `Vue.js` (while package is simply named `vue`). So the question is if we should keep it consistent or have correct name here :see\_no\_evil:

Comment (f1ames) [reply README.md:53]: Ok, looking more in some other places it seems using `Vue.js` is a correct approach.

**Comment by f1ames** [comment]
I was wondering about target branch, but since `major` is our main (and visible by default) GH branch, let's merge it there :+1:

File: README.md (hunk 1 above)
Comment (Dumluregn) [reply README.md:53]: Yeah, I wasn't sure about it too, but seems they are adding `.js` as a part of official name 🤔

**Pull request merged**